app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
i'm actually okay with  'subscription' and 'content'.  @ehelms do you have thoughts?